### PR TITLE
Anya/create profile for existing user backend

### PR DESCRIPTION
**Pull request by awesomeann**
## Describe your changes

I have created Profile app with Profile model in it. There are currently two views:

1. ProfileList View - has get and post methods
2. ProfileDetail - has GET, PUT, DELETE methods and permissions

I have created serializers for both views and urls end points.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## What kind of feedback do you need?

Check the code 

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have recommended where to start and how to proceed with the review

### CODE REVIEW

File: DiversiTech/profiles/models.py
Comment (blakerach1) [inline DiversiTech/profiles/models.py:1]: Include field that has choice argument passed in to deal with contact preference choices

Comment (blakerach1) [reply DiversiTech/profiles/models.py:1]: Let's assume that the nomination process is a nice to have. So, the profile model needs to assume that a user already existing. Therefore, let's remove redundant fields from the model - specifically username, first_name and last_name.

File: DiversiTech/profiles/serializers.py
Comment (blakerach1) [inline DiversiTech/profiles/serializers.py:1]: Update to reflect comments above

**Review by shayzimm — APPROVED** [review]

**Review by ktechgau — APPROVED** [review]

**Comment by blakerach1** [comment]
Anya has instructed me to merge this PR

**Pull request merged**